test/sys/test_socket.rs

```diff
@@ -301,14 +301,36 @@ fn test_scm_credentials() {
 /// Ensure that we can send `SCM_CREDENTIALS` and `SCM_RIGHTS` with a single
 /// `sendmsg` call.
 #[cfg(any(target_os = "android", target_os = "linux"))]
+// qemu's handling of multiple cmsgs is bugged, ignore tests on non-x86-64
+#[cfg_attr(not(target_arch = "x86_64"), ignore)]
```

Comment:
What about i386?  That doesn't use qemu on FreeBSD, and I don't think it does on Linux either.